tests/scr-readingsystem-support_iframe_svg/OPS/iframe_content.svg

```diff
@@ -0,0 +1,80 @@
+<svg version="1.1" viewBox="0 0 300 200" xmlns="http://www.w3.org/2000/svg">
+  <script>//<![CDATA[
```

Comment:
Why do only 3 of these files have CDATA? Fine if appropriate, but I'm unfamiliar with the need/distinction.

Reply:
(Oh, because they're SVGs!)